bodhi/tests/server/functional/base.py

```diff
@@ -13,23 +13,14 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
 import os
-import unittest
 
 from webtest import TestApp
-from sqlalchemy import create_engine
-from sqlalchemy import event
-from sqlalchemy.orm import scoped_session, sessionmaker
-from zope.sqlalchemy import ZopeTransactionExtension
 
-from bodhi.server import main, log
-from bodhi.tests.server import populate
-from bodhi.server.models import (
-    Base,
-)
+from bodhi.server import main
+from bodhi.tests.server.base import BaseTestCase, DB_NAME, DB_PATH
+
 
 FAITOUT = 'http://209.132.184.152/faitout/'
```

Comment:
The URL can be changed to the one I specified above :)

And I guess this answers my question :)